core/src/com/unciv/logic/map/TileMap.kt

```diff
@@ -357,6 +357,22 @@ class TileMap {
         }
     }
 
+    companion object {
+        // Statically allocate the Vectors (in World coordinates)
+        // of the 6 clock directions for border and road drawing in TileGroup 
+        private val clockToHexVectors: Map<Int,Vector2> = mapOf(
+            2 to HexMath.hex2WorldCoords(Vector2(0f, -1f)),
+            4 to HexMath.hex2WorldCoords(Vector2(1f, 0f)),
+            6 to HexMath.hex2WorldCoords(Vector2(1f, 1f)),
+            8 to HexMath.hex2WorldCoords(Vector2(0f, 1f)),
+            10 to HexMath.hex2WorldCoords(Vector2(-1f, 0f)),
+            12 to HexMath.hex2WorldCoords(Vector2(-1f, -1f))
+        )
```

Comment:
This should go in HexMath

Reply:
Sounds plausible. Placement was steered by 'let's plonk this in right next to where my screen is currently looking at' - but then the whole routine might be misplaced as the only benefit it gets from `this` is `getNeighborTileClockPosition` having access to map dimensions... Needs a bit of thought.